YogaKit/Source/YGLayout.h

```diff
@@ -10,9 +10,25 @@
 #import <UIKit/UIKit.h>
 #import <yoga/YGEnums.h>
 
+@class YGLayout;
+
+@protocol YGLayoutEntity <NSObject>
+@property (nonatomic, readonly, strong) YGLayout *yoga;
```

Comment:
Personally, I dislike having to use `synthesize`, I would rather make the protocol only contain methods. As a result, it leaves the implementation details up to the implementor. Maybe they don't want to use a property.

I also like the contract a little better. If someone wants to implement the protocol and use the property it might look something like this:

```
@interface SomeClass <YGLayoutEntity>
@property (nonatomic, readonly, strong) YGLayout *yoga;
...
@end
```
To me, this is much easier to read than just getting an object like this:

```
@interface SomeClass: NSObject <YGLayoutEntity>
@end
```

@hartbit what do you think?